### ISSUE TITLE
Update 'About' Acknowledgements

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/gcd#206]
The 'About' acknowledgements stuff is out of date. 

Please break into two sections. 
1. In the 'Funding' § grab content from [Financial Support](http://gcd.riverscapes.xyz/acknowledgements.html)
2. In the **Development Team** §, grab content from [Who is Page](http://gcd.riverscapes.xyz/who.html#development-team), just from the Development team minus the last sentence about Riverscapes Consortium (@philipbaileynar wordsmith first as you see fit).
 
![aboutupdates](https://user-images.githubusercontent.com/9723905/38341877-1cb313d6-3838-11e8-968f-064013bf1e5b.png)

@jb10016 and @philipbaileynar can you please have a look at both: [Who is Page](http://gcd.riverscapes.xyz/who.html#development-team) &  [Financial Support](http://gcd.riverscapes.xyz/acknowledgements.html) and wordsmith as you see fit. @jb10016 you can do this by simply clicking on edit in https://github.com/Riverscapes/gcd/blob/master/docs/who.md and
 https://github.com/Riverscapes/gcd/blob/master/docs/acknowledgements.md respectively. We want to get these pages done now so that release for workshop has proper about. @jb10016 in particular, please put in any acknowledgements you think might be necessary (for example in http://gcd.riverscapes.xyz/who.html#user-support-and-testing section of all the GeoTERM partners and such).

**Comment by jb10016** [comment]
Looks good! I'm tied up today, but will look at this as soon as I'm in the UK (on Sunday).  I think it would be good to name the regional councils directly and I'll have a tinker with a couple of bits, but this is a nice background context for the work.  Nice one.

**Comment by joewheaton** [comment]
Thanks James. If you have logos and URLs for them, send them my way or dump
them in /docs/assets/images/logos and I'll take care of it.

_____________________
Dr. Joseph M. Wheaton
435-554-1247 (direct) | http://www.joewheaton.org

**Comment by jb10016** [comment]
Logos here ... I think we might need to add them in as financial contributors to GCD7 as they have provided matching costs as part of the NERC grant ... 
![otagoregionalcouncil_logo](https://user-images.githubusercontent.com/35811609/38342421-66a22f46-38d2-11e8-8f80-6cd1f75f115f.png)
![environmentcanterbury_logo](https://user-images.githubusercontent.com/35811609/38342423-68223096-38d2-11e8-9061-e20615748cd0.png)
![hawkesbayregionalcouncil_logo](https://user-images.githubusercontent.com/35811609/38342424-69535620-38d2-11e8-9c9f-19fbb6b19b65.jpg)
![sepa_logo](https://user-images.githubusercontent.com/35811609/38342426-6ac7fbb4-38d2-11e8-9c77-408ee2756077.png)

**Comment by joewheaton** [comment]
Perfect. I will put these in.

_____________________
Dr. Joseph M. Wheaton
435-554-1247 (direct) | http://www.joewheaton.org

**Comment by jb10016** [comment]
No worries - I've put them (along with a Waikato logo) in the /logos folder too ...

**Comment by jb10016** [comment]
I've tinkered a bit with the Who Is page ... feel free to change it back again!

**Comment by joewheaton** [comment]
Looks good. Thanks.

_____________________
Dr. Joseph M. Wheaton
435-554-1247 (direct) | http://www.joewheaton.org

**Comment by philipbaileynar** [comment]
I've updated the about form in both the AddIn and Standalone. The old approach of using multiple text and image boxes was excruciating and time consuming to work with. So I've switched the acknowledgements to a web control that shows the contents of a special, hidden, page in the GCD documentation web site.

This way we can tweak and improve the acknowledgements section more quickly and have more control over the graphics. It does mean that this section of the form requires an internet connection. But if that's prohibitive then we can include a copy of this page within the software and refer to it locally.

Note that the page used is a **copy** of the acknowledgements. i.e. if you want to update what appears in the form, change the special copy not the one visible online within the documentation site.

* [Regular GCD acknowledgements page](http://gcd.riverscapes.xyz/acknowledgements.html)
* [special, hidden copy of acknowledgements used by GCD About form](http://gcd.riverscapes.xyz/dotnetack.html)


![2018-04-11_164140](https://user-images.githubusercontent.com/8645379/38648782-ee6db9c0-3da7-11e8-8449-1be0a94674d2.png)
